config/externalnamenottested.go

```diff
@@ -952,4 +952,121 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"aws_redshiftserverless_usage_limit": config.IdentifierFromProvider,
 	// Redshift Serverless Workgroups can be imported using the workgroup_name
 	"aws_redshiftserverless_workgroup": config.ParameterAsIdentifier("workgroup_name"),
+
+	// resourceexplorer2
+	//
+	// Resource Explorer indexes can be imported using the arn
+	// Example: arn:aws:resource-explorer-2:us-east-1:123456789012:index/6047ac4e-207e-4487-9bcf-cb53bb0ff5cc
+	"aws_resourceexplorer2_index": config.IdentifierFromProvider,
+	// Resource Explorer views can be imported using the arn
+	// Example: arn:aws:resource-explorer-2:us-west-2:123456789012:view/exampleview/e0914f6c-6c27-4b47-b5d4-6b28381a2421
+	"aws_resourceexplorer2_view": config.IdentifierFromProvider,
+
+	// rolesanywhere
+	//
+	// aws_rolesanywhere_profile can be imported using its id
+	"aws_rolesanywhere_profile": config.IdentifierFromProvider,
+	// aws_rolesanywhere_trust_anchor can be imported using its id
+	"aws_rolesanywhere_trust_anchor": config.IdentifierFromProvider,
+
+	// route53
+	//
+	// Route 53 Resolver configs can be imported using the Route 53 Resolver config ID
+	"aws_route53_resolver_config": config.IdentifierFromProvider,
+
+	// rum
+	//
+	// Cloudwatch RUM App Monitor can be imported using the name
+	"aws_rum_app_monitor": config.NameAsIdentifier,
+	// Cloudwatch RUM Metrics Destination can be imported using the id
+	"aws_rum_metrics_destination": config.ParameterAsIdentifier("app_monitor_name"),
+
+	// s3
+	//
+	// S3 Storage Lens configurations can be imported using the account_id and config_id, separated by a colon (:)
+	"aws_s3control_storage_lens_configuration": config.IdentifierFromProvider,
+
+	// sagemaker
+	//
+	// Models can be imported using the id
+	"aws_sagemaker_servicecatalog_portfolio_status": config.IdentifierFromProvider,
+	// SageMaker Spaces can be imported using the id
+	// Example: arn:aws:sagemaker:us-west-2:123456789012:space/domain-id/space-name
+	"aws_sagemaker_space": config.IdentifierFromProvider,
+
+	// scheduler
+	//
+	// Schedules can be imported using the combination group_name/name
+	"aws_scheduler_schedule": config.IdentifierFromProvider,
+	// Schedule groups can be imported using the name
+	"aws_scheduler_schedule_group": config.IdentifierFromProvider,
```

Comment:
It seems that we can use the following configuration, what do you think?

```suggestion
	"aws_scheduler_schedule_group": config. NameAsIdentifier,
```

Reply:
Fixed